Source/DafnyCore/Dafny.atg

```diff
@@ -964,16 +964,19 @@ IdentTypeOptional<out BoundVar var>
      }; .)
   .
 
-TypeIdentOptional<out RangeToken/*!*/ range, out Name/*!*/ identName, out Type/*!*/ ty, out bool isGhost, out Expression defaultValue, out bool isNameOnly>
+TypeIdentOptional<out RangeToken/*!*/ range, out Name/*!*/ identName, out Type/*!*/ ty, 
+  out bool isGhost, out Expression defaultValue, out bool isNameOnly, out Attributes attributes>
 = (.Contract.Ensures(Contract.ValueAtReturn(out range)!=null);
      Contract.Ensures(Contract.ValueAtReturn(out ty)!=null);
      Contract.Ensures(Contract.ValueAtReturn(out identName)!=null);
      IToken nameToken = null; ty = new BoolType()/*dummy*/; isGhost = false;
      IToken nameonlyToken = null;
+     attributes = null;
      identName = null;
      defaultValue = null;
   .)
   (. var beforeStartToken = t; .)
+  { Attribute<ref attributes> }
```

Comment:
Indeed, I validate putting these before the other modifiers like ghost or older. We should do the same and put attributes before function and methods keywords as well, but let's keep it for another time.